cmd/openvdc/cmd/log.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	mlog "github.com/ContainX/go-mesoslog/mesoslog"
+	log "github.com/Sirupsen/logrus"
+	"github.com/axsh/openvdc/cmd/openvdc/internal/util"
+	"github.com/spf13/cobra"
+)
+
+var tail bool
+
+func init() {
+	logCmd.Flags().BoolVarP(&tail, "tail", "t", false, "Tail log output instead of just printing it once.")
+}
+
+var logCmd = &cobra.Command{
+	Use:   "log [Instance ID]",
+	Short: "Print logs of an instance",
+	Long:  "Print logs of an instance",
+	Example: `
+	% openvdc log i-xxxxxxx
+	`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			log.Fatalf("Please provide an Instance ID.")
+		}
+
+		instanceID := "VDC_" + args[0]
+
+		split := strings.Split(util.MesosMasterAddr, ":")
+		mesosMasterAddr := split[0]
+		mesosMasterPort, err := strconv.Atoi(split[1])
+
+		if err != nil {
+			log.Errorln("Couldn't convert string to int") //  <--- lol
```

Comment:
This should include the actual message from `err`. Same goes for the logs further down this code.

I also quickly tried to look up the `log.Errorln` function but couldn't find it right away. Does it exit with a non-zero exit code? The way it's named, I'd expect it to log the error and then continue executing the code below but this type of error strikes me as something that should be fatal.